develop/tutorials/articles/260-forms/01-form-field-types/01-creating-form-field-types.markdown

```diff
@@ -411,14 +403,13 @@ All that's left to do is create the `config.js` file:
         );
     })();
 
-This file is entirely boilerplate, and you'll never need anything different if
-you follow the conventions described above. In fact, if you use Blade CLI to
-generate a field type module, you will get this file ready to use. So what is the `config.js` file for? It's a JavaScript file that defines the
-dependencies of the declared JavaScript components (`requires...`), and where
-the files are located (`path...`). The `config.js` is used by the Alloy loader
-when it satisfies dependencies for each JavaScript component. For more
-information about the Alloy loader see the [tutorial on its
-usage](/develop/tutorials/-/knowledge_base/7-0/liferay-amd-module-loader).
+This file is entirely boilerplate. In fact, if you use Blade CLI to generate a
+field type module, you won't need to touch this file. Functionally, it's a
+JavaScript file that defines the dependencies of the declared JavaScript
+components (`requires...`), and where the files are located (`path...`). The
+`config.js` is used by the Alloy loader when it satisfies dependencies for each
+JavaScript component. For more information about the Alloy loader see the
+[tutorial on its usage](/develop/tutorials/-/knowledge_base/7-0/loading-amd-modules-in-liferay).
```

Comment:
This current link redirects to no article, but I realized that if you replace the 7-0 by 7-1 it works. 😉